src/net.rs

```diff
@@ -98,17 +98,20 @@ impl Clone for Box<NetworkStream + Send> {
 }
 
 impl NetworkStream {
```

Comment:
It was actually because I couldn't use them in the tests in the `client/mod.rs` file.

There should already be a public API, using `std::any::Any`, right?